ui/src/dashboards/components/Ranger.js

```diff
@@ -0,0 +1,99 @@
+import React, {PropTypes} from 'react'
+import _ from 'lodash'
+
+// TODO: add logic for for Prefix, Suffix, Scale, and Multiplier
+const Ranger = ({onSetRange, onSetLabel, axes}) => {
```

Comment:
If we keep its name as `Ranger`, then everytime I see this component in a patch I will continue to think:

![norris](https://i.makeagif.com/media/9-29-2015/OS_aF-.gif)